src/components/Navbar.tsx

```diff
@@ -72,64 +72,64 @@ const Navbar: React.FC<NavbarProps> = () => {
                     {!isAuthenticated ? (
                         <>
                             <button
-                                className="text-white no-underline font-normal transition duration-300 hover:text-[#252F70]"
+                                className="text-primary no-underline font-medium transition duration-300 hover:text-[#252F70]"
                                 onClick={() => handleNavigation('/')}
                             >
                                 HOME
                             </button>
                             <button
-                                className="text-white no-underline font-normal transition duration-300 hover:text-[#252F70]"
+                                className="text-primary no-underline font-medium transition duration-300 hover:text-[#252F70]"
                                 onClick={() => handleNavigation('#services')}
                             >
-                                SERVICES
+                                ABOUT
                             </button>
                             <button
-                                className="text-white no-underline font-normal transition duration-300 hover:text-[#252F70]"
+                                className="text-primary no-underline font-medium transition duration-300 hover:text-[#252F70]"
                                 onClick={() => handleNavigation('#about')}
                             >
-                                ABOUT US
+                                SERVICES
                             </button>
                             <button
-                                className="text-white no-underline font-normal transition duration-300 hover:text-[#252F70]"
+                                className="text-primary no-underline font-medium transition duration-300 hover:text-[#252F70]"
                                 onClick={() => handleNavigation('#contacts')}
                             >
-                                CONTACTS
+                                CONTACTS 
                             </button>
                             <button
-                                className="text-white no-underline font-normal transition duration-300 hover:text-[#252F70]"
+                                className="text-primary no-underline font-medium transition duration-300 hover:text-[#252F70]"
                                 onClick={() => handleNavigation('/signup')}
                             >
-                                CREATE ACCOUNT
```

Comment:
do not delete this tab.

Reply:
try to click also the tabs if it is redirecting to its intended page.

Reply:
resolve the issue already